src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -574,6 +591,457 @@ export class MicroMetaTabView extends React.PureComponent {
 
 }
 
+let microMetaDependencies = null;
+const MicroMetaSummaryTabViewFRef = React.forwardRef((props, ref) => <MicroMetaSummaryTabView {...props} forwardRef={ref} />);
+export class MicroMetaSummaryTabView extends React.PureComponent {
+    
+    static getTabObject(props, width) {
+        const { ref } = props;
+
+        return {
+            'tab': <span><i className="icon icon-list-alt fas icon-fw" /> Hardware Summary</span>,
+            'key': 'hardware-summary',
+            'content': <MicroMetaSummaryTabViewFRef {...props} width={width} />
+        };
+    }
+
+    static facetsFromMicroscope(microscope) {
+        const categoryObj = {};
+        _.forEach(microscope.components, function (component) {
```

Comment:
(For future) Using native version of e.g. Array.forEach (+ map, Object.keys, etc.) is slightly preferred, in case we want to migrate away from using underscore at some point. Previously, some browsers didn't implement their own .forEach, .map, .reduce, etc. in their JS engines/implementations, but this was 5-10+ yrs ago. There's some other handy underscore functions which we use still though (zip, throttle, debounce, pluck, ...), though theoretically we could replace with lighter-weight subset of lodash or something at some point in future.